src/NuGet/Microsoft.CodeAnalysis.CSharp.nuspec

```diff
@@ -14,6 +14,7 @@
     </description>
     <dependencies>
       <dependency id="Microsoft.CodeAnalysis.Common" version="[$version$]" />
+      <dependency id="Humanizer.Core" version="$HumanizerCoreVersion$" />
```

Comment:
This is the compiler package, it's definitely not being used there.

Reply:
removed